oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/annotation/AnnotationListener.java

```diff
@@ -16,14 +16,16 @@
  *
  */
 
-package org.apache.skywalking.oap.server.core.analysis.indicator.define;
+package org.apache.skywalking.oap.server.core.annotation;
+
+import java.lang.annotation.Annotation;
 
 /**
  * @author peng-yongsheng
  */
-public class IndicatorDefineLoadException extends Exception {
+public interface AnnotationListener {
+
+    Class<? extends Annotation> annotation();
 
-    public IndicatorDefineLoadException(String message, Throwable cause) {
-        super(message, cause);
-    }
+    void ownerClass(Class aClass);
```

Comment:
I feel a little strange about `ownerClass` method name. This is a method of `*Listenser`,  owner makes me think it is the owner of the listener. Clearly, it isn't. 

So I proposal, you should name this as something like `notify`, which should more make sense.

Reply:
notify is a key word.

Reply:
No, it isn't.